src/index.html

```diff
@@ -13,6 +13,56 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <block class="block">
```

Comment:
The `<block>` element is not a standard HTML element. Consider using a `<div>` or `<section>` instead for better semantic structure.